daemon/cmd/daemon.go

```diff
@@ -343,12 +345,7 @@ func (d *Daemon) restoreCiliumHostIPs(ipv6 bool, fromK8s net.IP) {
 	}
 
 	restoredIP := node.RestoreHostIPs(ipv6, fromK8s, fromFS, cidrs)
-	if err := removeOldRouterState(ipv6, restoredIP); err != nil {
-		log.WithError(err).Warnf(
-			"Failed to remove old router IPs (restored IP: %s) from cilium_host. Manual intervention is required to remove all other old IPs.",
-			restoredIP,
```

Comment:
I think it can still be nice to know what the restored IP is in the log msg so that the user can know which old IPs to remove

Reply:
No loss here, as we moved the log message further up the stack. 
Hence failed deletions will still log the respective host IPs that could not be deleted.